src/styles/blocks/calendar.scss

```
@@ -0,0 +1,46 @@
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  gap: $cell-gap;
+  max-width: 7 * ($cell-size + $cell-gap);
+
+  &__cell {
+    position: relative;
+    @include square($cell-size);
+    border: 1px solid #000;
```

don't forget to create variables for all colors

```suggestion
    border: 1px solid $c-black;
```